packages/roosterjs-editor-core/lib/editor/Editor.ts

```diff
@@ -740,7 +740,7 @@ export default class Editor implements IEditor {
      * @returns Default format object of this editor
      */
     public getDefaultFormat(): DefaultFormat {
-        return this.getCore().lifecycle.defaultFormat;
+        return this.getCore().lifecycle.defaultFormat!;

```

Comment:
If defaultFormat can really be null, we should allow this function returns null, or return an empty object "{}"